features/image-optimization/README.md

```diff
@@ -11,3 +11,4 @@ The following features related to image optimization can be found in your dashbo
 3. [Chroma Subsampling](chroma-subsampling.md)
 4. [Metadata, Color Profile and Orientation](metadata-color-profile-and-orientation.md)
 5. [PNG Compression](png-compression.md)
+6. [SVG to raster format conversion](svg-to-raster-format-conversion.md)
```

Comment:
This should go under the `Image transformations` header imo. Converting SVGs to Raster formats isn't really an optimization.

Reply:
Moved the section to README of image-transformations

Please check this screenshoot for overall diff in the text

<img width="1393" alt="Screenshot 2023-12-29 at 10 59 00 AM" src="https://github.com/imagekitio/docs-keep-a-changelog/assets/80034794/aa69547e-6e4e-4478-9803-33c8067fc988">

Reply:
I meant a different page for this, but under the `Image transformations` main heading